packages/components/src/ListView/Items/items.snapshot.test.js

```diff
@@ -32,4 +33,12 @@ describe('Items', () => {
 		// then
 		expect(toJson(wrapper)).toMatchSnapshot();
 	});
+
+	it('should render without toggleAll checkbox', () => {
+		// when
+		const wrapper = mount(<Items {...props} toggleAllShown={false} />);
+
+		// then
+		expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
The snapshot is pretty big, so it's hard to check if it's correct.
Can you replace it with a wrapper.find('.tc-listview-toggle-all') and you check if you find any ?